index.src.html

```diff
@@ -761,18 +722,34 @@ <h4 id="algorithm-request" algorithm>Request a `Credential`</h4>
 
     5.  Let |origin| be the [=current settings object=]'s [=environment settings object/origin=].
 
-    6.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
-        ancestors=], and `false` otherwise.
+    6.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with
+        its ancestors=], and `false` otherwise.
 
-    7.  Run the following steps [=in parallel=]:
+    7.  Let |key| be the result of [=map/getting the key=] of |options|.
+    
+        Issue: do we need to in this algo get the [=set=] of
+        |options|' <a>relevant credential interface objects</a> and ensure the latter as well
+        as |options| contain only 1 item/member referring to the same credential type, as is
+        done in steps 5, 6, 7 of [[#algorithm-create]].
```

Comment:
I don't think we do, because we explicitly wanted to support picking one of several kinds of credentials. We haven't actually done that yet, and maybe it turns out not to make sense to do so, but for the moment, I'd leave this out.